operations/patch.go

```diff
@@ -116,18 +119,24 @@ func Patch() cli.Command {
 				return err
 			}
 
-			diffData, err := loadGitData(ref.Branch, params.Ref, "", useMbox, args...)
-			if err != nil {
-				return err
-			}
-
 			if params.Description == "" {
 				params.Description, err = getDefaultDescription()
 				if err != nil {
 					grip.Error(err)
 				}
 			}
 
+			diffData, err := loadGitData(ref.Branch, params.Ref, "", params.PreserveCommits, args...)
+			if err != nil {
+				return err
+			}
+			if !params.PreserveCommits {
```

Comment:
my plan was to warn the user their uncommitted changes will be ignored (earlier) and make an mbox patch out of what's already been committed.

Reply:
But the only case where this would happen is if they have the uncommitted changes flag in their configuration file, which gives a warning but users are sometimes odd and they might not see it (especially when it doesn't require a prompt). So would this still allow users to use merge_patch to merge the committed changes?

I think either this check needs to be strict (don't continue with  PreserveCommits && UncommittedChanges) and/or make the warning require a yes/no confirmation.

Reply:
done